docs/application/web/tutorials/overview.md

```diff
@@ -10,6 +10,8 @@ The introduction to Web applications documentation provides overall information
 
 - [Security and API Privileges](sec-privileges.md)
 
+- [Security and API Privileges for Old Apps](old-versioned-sec-privileges.md)
```

Comment:
Suggestion: Old apps does not convey the right meaning. It will be good to specify the version.

Suggest rephrasing to:

[Security and API Privileges for Apps with API Version 4.0 or Earlier](old-versioned-sec-privileges.md)

or

[Security and API Privileges for 4.0 Apps or Earlier](old-versioned-sec-privileges.md)

Please verify and update.

Reply:
done